todo-backend/README.md

```
@@ -251,19 +254,190 @@ Creates a new task
+        "deferredTill": "<datetime> | null"
+    } | null,
+    "dueAt": "<datetime> | null",
+    "startedAt": "<datetime> | null"
+  }
+  ```
+
+- **Success Response:**
+
+  - **Code:** 200
+  - **Content:**
+    ```json
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)